examples/src/intTest/java/ca/nrc/cadc/UserStorageBrowserPage.java

```diff
@@ -193,38 +193,42 @@ public UserStorageBrowserPage(final WebDriver driver) throws Exception
         super(driver);
         this.driver = driver;
 
-        if ( elementExists(By.xpath("//*[@id=\"main_section\"]")) )
-        {
-            waitForStorageLoad();
-        }
+//        if ( elementExists(By.xpath("//*[@id=\"main_section\"]")) )
+//        {
+//            waitForStorageLoad();
+//        }
 
         PageFactory.initElements(driver, this);
     }
 
 
+
+
     // Transition functions
-    public void clickButton(String promptText) throws Exception
+    public UserStorageBrowserPage clickButton(String promptText) throws Exception
     {
         final By buttonBy =
                 xpath("//button[contains(text(),\"" + promptText + "\")]");
         waitForElementClickable(buttonBy);
         click(buttonBy);
+        return new UserStorageBrowserPage(driver);
     }
 
-    public void clickButtonWithClass(String promptText, String className) throws
+    public UserStorageBrowserPage clickButtonWithClass(String promptText, String className) throws
                                                                           Exception
     {
         final By buttonWithClassBy =
                 xpath("//button[contains(@class, '" + className
                       + "') and contains(text(),'" + promptText + "')]");
         waitForElementClickable(buttonWithClassBy);
         click(buttonWithClassBy);
+        return new UserStorageBrowserPage(driver);
     }
 
-    public void enterSearch(final String searchString) throws Exception
+    public UserStorageBrowserPage enterSearch(final String searchString) throws Exception
```

Comment:
Entering a search doesn't refresh the page, so having a void function should be fine.  The function should wait for the result to come back.